actionmailbox/app/controllers/action_mailbox/ingresses/amazon_ses/inbound_emails_controller.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module ActionMailbox
+  # Ingests inbound emails from Amazon SES/SNS and confirms subscriptions.
```

Comment:
This should be on top of the `AmazonSes` class, not the `Ingresses` module.

Reply:
Fixed.